From 648dee9a042d08c669537d72b5bdcd31b99f6c74 Mon Sep 17 00:00:00 2001
From: Phil Sutter <psutter@redhat.com>
Date: Fri, 22 Jul 2016 14:53:00 +0200
Subject: [PATCH] man: ip-link: Document query_rss option

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1264146
Upstream Status: iproute2.git commit 8e45e44b7923a

commit 8e45e44b7923a30c97ea50e43f72db6688c6175e
Author: Phil Sutter <phil@nwl.cc>
Date:   Fri Jun 10 16:39:50 2016 +0200

    man: ip-link: Document query_rss option

    Doc text shamelessly stolen from the introducing commit's message
    (6c55c8c4617c5 ['ip link set vf: Added "query_rss" command']).

    Signed-off-by: Phil Sutter <phil@nwl.cc>
---
 man/man8/ip-link.8.in | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/man/man8/ip-link.8.in b/man/man8/ip-link.8.in
index d98fe91..33172e5 100644
--- a/man/man8/ip-link.8.in
+++ b/man/man8/ip-link.8.in
@@ -108,6 +108,8 @@ ip-link \- network device configuration
 .br
 .RB "[ " spoofchk " { " on " | " off " } ]"
 .br
+.RB "[ " query_rss " { " on " | " off " } ]"
+.br
 .RB "[ " state " { " auto " | " enable " | " disable " } ]"
 .br
 .RB "[ " trust " { " on " | " off " } ] ]"
@@ -1028,6 +1030,9 @@ parameter must be specified.
 .BI spoofchk " on|off"
 - turn packet spoof checking on or off for the specified VF.
 .sp
+.BI query_rss " on|off"
+- toggle the ability of querying the RSS configuration of a specific VF. VF RSS information like RSS hash key may be considered sensitive on some devices where this information is shared between VF and PF and thus its querying may be prohibited by default.
+.sp
 .BI state " auto|enable|disable"
 - set the virtual link state as seen by the specified VF. Setting to auto means a
 reflection of the PF link state, enable lets the VF to communicate with other VFs on
-- 
1.8.3.1

